packages/flutter/lib/src/material/stepper.dart

```diff
@@ -204,12 +204,20 @@ class Stepper extends StatefulWidget {
     this.controlsBuilder,
     this.elevation,
     this.margin,
+    this.connectorColor,
+    this.connectorThickness = 2.0,
   }) : assert(steps != null),
        assert(type != null),
        assert(currentStep != null),
        assert(0 <= currentStep && currentStep < steps.length),
        super(key: key);
 
+  /// The thickness of the connecting lines.
+  final double? connectorThickness;
+
+  /// The color of the connection lines.
+  final MaterialStateProperty<Color>? connectorColor;
```

Comment:
I think this should be `MaterialStateProperty<Color?>?` instead, so that you can fall back to a default if they don't set it.

Reply:
Take a look at how `Checkbox` evaluates its `fillColor` for an example of the pattern: https://github.com/flutter/flutter/blob/94fefaa49d86fde856146c05d6817382a2fe7b3c/packages/flutter/lib/src/material/checkbox.dart#L435